docs/eln/about.mdx

```diff
@@ -6,14 +6,14 @@ sidebar_position: 9
 
 # Chemotion ELN
 
-The Chemotion ELN was initiated by the working group of Stefan Bräse at the Institute of Organic Chemistry (IOC)/Institute of Biological and Chemical Systems (IBCS) of Karlsruhe Institute of Technology (KIT). Currently, the software is developed by a team working at the the north campus of the KIT and in colaberation with developers at the universities Jena, Braunschweig, Halle and RWTH Aachen.
+The Chemotion ELN was initiated by the working group of Stefan Bräse at the Institute of Organic Chemistry (IOC)/Institute of Biological and Chemical Systems (IBCS) of Karlsruhe Institute of Technology (KIT). Currently, the software is developed by a team working at the the north campus of the KIT and in collaboration with developers at the universities Jena, Braunschweig, Halle and RWTH Aachen.
 The Chemotion ELN is developed for chemists and other scientists working in related fields.
-
-We would like to thank many other institutions and developers who contributed code, ideas and finished subcomponents. The Chemotion ELN could only come into being with the support of developers from NinjaConcept and Cubuslab. Further sub-components are available through the cooperation with NMRdb and SciFinder.
+docs/eln/about.mdx
```

Comment:
@lemondi93 these lines (11, 14 and 16) are a typo I guess?

Reply:
yes they